api/v3/Order.php

```diff
@@ -92,8 +92,9 @@ function civicrm_api3_order_create($params) {
   $entityIds = [];
   $contributionStatus = CRM_Utils_Array::value('contribution_status_id', $params);
   if ($contributionStatus !== 'Pending' && 'Pending' !== CRM_Core_PseudoConstant::getName('CRM_Contribute_BAO_Contribution', 'contribution_status_id', $contributionStatus)) {
-    CRM_Core_Error::deprecatedFunctionWarning('Creating a Order with a status other than pending is deprecated. Currently empty defaults to "Completed" so as a transition not passing in "Pending" is deprecated');
+    CRM_Core_Error::deprecatedFunctionWarning("Creating a Order with a status other than pending is deprecated. Currently empty defaults to 'Completed' so as a transition not passing in 'Pending' is deprecated. You can chained payment creation like - civicrm_api3('Order', 'create', ['blah' => 'blah', 'contribution_status_id' => 'Pending', 'api.Payment.create => ['total_amount' => 5]]");
```

Comment:
Grammar: "You can chained payment creation like -"   Maybe "You can chain payment creation - for example:"

Reply:
thanks - fixed